lib/ansible/module_utils/f5_utils.py

```diff
@@ -140,6 +140,10 @@ def fq_list_names(partition,list_names):
 
 # New style
 
+from abc import ABCMeta, abstractproperty
+from ansible.compat.six import with_metaclass
```

Comment:
No, these are fine, because the code in plugins is executed on the control node.

However, code in module_utils and module are typically executed on a remote (except for local action). So the code must be transfered to the remote target. The module and module_utils are transfered, that is why module_utils.six must be used with in modules and module_utils